src/status_im/events.cljs

```diff
@@ -1015,6 +1015,11 @@
  (fn [cofx _]
    (hardwallet/get-application-info cofx nil nil)))
 
+(handlers/register-handler-fx
+ :hardwallet/save-key-uid-to-account
```

Comment:
you can now declare this event by adding `{:events  :hardwallet/save-key-uid-to-account]}` in the attribute map of `hardwallet/save-key-uid-to-account` (and use an event name that reflects what happened rather than the action you want to take

Reply:
@yenda right, i'm aware that fx/defn now have this feature. I can start using it here.

I'm trying to follow event name conventions, but for this particular event i'm not sure i can come up with good name of what's happened because chain of events should happen in order to trigger this like: `on-get-keys-success -> on-get-application-info-success -> save-key-uid-to-account`

Reply:
if this is the chain of event without any user action you might as well not use an event for the intermediate events that don't need up to date cofx and don't produce effects themselves, I'm doing this for swarm for instance, where only errors in the chain dispatch events.

Reply:
it might be too many changes that you don't want to make in this PR that's ok, but think about it for the future, if you have these sequences of event just encapsulate them with multiple levels of callback

Reply:
Good suggestions in general, but for this particular case i'd prefer to leave it as is because this code will removed before release.